static/lang/re-en.json

```diff
@@ -4367,7 +4367,8 @@
                 },
                 "UnifiedMagicalTheory": {
                     "Prompt": "Select a 1st-level wizard feat."
-                }
+                },
+                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[@item.level[fire]] damage. If the creature summoned has the acid, cold, electricity, sonic, or spirit trait, you can deal damage of that type instead."
```

Comment:
Let's also move this under a spellshape section, so it is `PF2E.SpecificRule.Spellshape.ExplosiveArrival`